### DISCUSSION TITLE
Bearshare

### DISCUSSION THREAD

**Discussion by sebastian-salmhofer** [engageub/InternetIncome#163] [General]
Hi everyone,
what are your experiences with Bearshare? I've been running it for a few days now with 30 nodes but didn't get a single cent. Are they still active?

**Comment by engageub** [comment]
Hello,

Bearshare, Bytelixir and Peer2Profit do not require apps if you have Proxies.

For Bearshare, please use the following URL to add your Proxies. You can raise a support ticket in their official discord channel if you don't have access to the following link. 

https://app.bearshare.app/proxied

Payments have been received from bearshare multiple times. 


Thank you

**Reply by sebastian-salmhofer** [reply]
Hi,

oh that's great. I read about it, but I couldn't find an option to add proxies in their interface, so I thought maybe they changed it. But it works with your link, thank you. Bytelixir I am already using with proxies and Peer2Profit doesn't take new people unfortunately.

**Reply by Raffy142** [reply]
Sorry, u have discord? Because i can’t find everywhere

**Reply by sebastian-salmhofer** [reply]
I do, sebastian.salmhofer_49287, but I don't use it much.

**Reply by engageub** [reply]
Hello,
Bearshare was cloudcollab earlier. The invitation link in their old discord channel might have expired, so its not accessible. 
Please find their discord channel link: https://discord.gg/BdK4PCnY

Thank you
